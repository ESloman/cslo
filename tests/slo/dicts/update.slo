var alpha = {"a": 1, "b": 2, "c": 3};
var alpha_more = {"d": 4, "e": 5, "f": 6};

print(alpha);
alpha.update(alpha_more);
print(alpha);
