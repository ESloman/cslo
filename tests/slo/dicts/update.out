dict[3]: {c: 3, a: 1, b: 2, }
dict[6]: {e: 5, f: 6, c: 3, d: 4, a: 1, b: 2, }
